Library/Homebrew/cask/blacklist.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Cask
+  module Blacklist
+    def self.blacklisted_reason(name)
+      case name
+      when /^adobe\-.+/i
+        "Adobe Casks are not actively maintained by Homebrew."
```

Comment:
I'm not 100% sure what the exact reason phrasing should be, so please let me know if I can be more accurate with these. For now, I'm going off of what was listed in https://github.com/Homebrew/homebrew-cask/issues/58040.